packages/tempus/src/Tempus_TimeStepControl_decl.hpp

```diff
@@ -152,12 +162,6 @@ class TimeStepControl
     virtual void setFinalIndex(int i) { finalIndex_ = i; isInitialized_ = false; }
     virtual void setMaxAbsError(Scalar e) { maxAbsError_ = e; isInitialized_ = false; }
     virtual void setMaxRelError(Scalar e) { maxRelError_ = e; isInitialized_ = false; }
-#ifndef TEMPUS_HIDE_DEPRECATED_CODE
-    virtual void setMinOrder(int)  { isInitialized_ = false; }
-    virtual void setInitOrder(int) { isInitialized_ = false; }
-    virtual void setMaxOrder(int)  { isInitialized_ = false; }
-#endif
-    virtual void setStepType(std::string s) { stepType_ = s; isInitialized_ = false; }
     virtual void setMaxFailures(int i) { maxFailures_ = i; isInitialized_ = false; }
     virtual void setMaxConsecFailures(int i) { maxConsecFailures_ = i; isInitialized_ = false; }
     virtual void setPrintDtChanges(bool b) { printDtChanges_ = b; isInitialized_ = false; }
```

Comment:
`setNumTimeSteps` only one not setting `isInitialized_` to false.

Reply:
I think it is being set in the *impl.hpp:458.